pkg/result/filter_test.go

```diff
@@ -611,162 +484,131 @@ func TestFilter(t *testing.T) {
-								Title:    "Bad Deployment",
-								Message:  "something bad",
-								Severity: dbTypes.SeverityLow.String(),
-								Status:   types.StatusFailure,
+								Type:    types.FindingTypeMisconfiguration,
+								Status:  types.FindingStatusIgnored,
+								Source:  "testdata/.trivyignore.yaml",
+								Finding: misconf2,
 							},
 						},
 					},
 					{
 						Target: "config.yaml",
-						Secrets: []ftypes.SecretFinding{
+						Secrets: []types.DetectedSecret{
+							secret1,
+						},
+						ModifiedFindings: []types.ModifiedFinding{
 							{
-								RuleID:    "generic-wanted-rule",
-								Severity:  dbTypes.SeverityLow.String(),
-								Title:     "Secret that should pass filter on rule id",
-								StartLine: 1,
-								EndLine:   2,
-								Match:     "*****",
+								Type:    types.FindingTypeSecret,
+								Status:  types.FindingStatusIgnored,
+								Source:  "testdata/.trivyignore.yaml",
+								Finding: secret2,
+							},
+							{
+								Type:    types.FindingTypeSecret,
+								Status:  types.FindingStatusIgnored,
+								Source:  "testdata/.trivyignore.yaml",
+								Finding: secret3,
 							},
 						},
 					},
 					{
+						Target: "LICENSE.txt",
 						Licenses: []types.DetectedLicense{
+							license2,
+						},
+						ModifiedFindings: []types.ModifiedFinding{
 							{
-								Name:       "GPL-3.0",
-								Severity:   dbTypes.SeverityLow.String(),
-								FilePath:   "usr/share/gcc/python/libstdcxx/v6/printers.py",
-								Category:   "restricted",
-								Confidence: 1,
+								Type:    types.FindingTypeLicense,
+								Status:  types.FindingStatusIgnored,
+								Source:  "testdata/.trivyignore.yaml",
+								Finding: license1,
 							},
 						},
 					},
 				},
 			},
 		},
 		{
-			name: "policy file",
+			name: "policy file for vulnerabilities",
 			args: args{
 				report: types.Report{
 					Results: types.Results{
 						{
 							Vulnerabilities: []types.DetectedVulnerability{
-								{
-									VulnerabilityID:  "CVE-2019-0001",
-									PkgName:          "foo",
-									InstalledVersion: "1.2.3",
-									FixedVersion:     "1.2.4",
-									Vulnerability: dbTypes.Vulnerability{
-										Severity: dbTypes.SeverityLow.String(),
-									},
-								},
-								{
-									// this vulnerability is ignored
-									VulnerabilityID:  "CVE-2019-0002",
-									PkgName:          "foo",
-									InstalledVersion: "1.2.3",
-									FixedVersion:     "1.2.4",
-									Vulnerability: dbTypes.Vulnerability{
-										Severity: dbTypes.SeverityLow.String(),
-									},
-								},
-								{
-									// this vulnerability is ignored
-									VulnerabilityID:  "CVE-2019-0003",
-									PkgName:          "foo",
-									InstalledVersion: "1.2.3",
-									FixedVersion:     "1.2.4",
-									Vulnerability: dbTypes.Vulnerability{
-										Severity: dbTypes.SeverityLow.String(),
-									},
-								},
+								vuln1,
+								vuln2, // ignored by policy
```

Comment:
We need to change severities or change comment:
```suggestion
								vuln2, // ignored by severity
```